H2MLauncher.Core/H2MLauncher.Core.csproj

```diff
@@ -10,6 +10,17 @@
   <ItemGroup>
     <PackageReference Include="CommunityToolkit.Mvvm" Version="8.2.2" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Storage\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="Storage\UserFavorites.json">
```

Comment:
This stores the configuration in the solution. When we release it we have to store the favorites somewhere else.. likely in the  `%localappdata%\BetterH2MLauncher` directory.

This is something which will have to be discussed first, so I will also tag @tobibodamer here.

Reply:
I agree. Favorites should not be included in the project, but stored in the appdata folder (see `Constants.LocalDir`)

Reply:
how should we call the folder ? BetterH2MLauncher ?

Reply:
It already exists. Just use the LocalDir constant from the `Constants` class